packages/libs/react-components/src/components/Card/styles.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint @kadena-dev/typedef-var: 0 */
+
+import { styled } from '../../styles';
+
+export enum CardColors {
+  Default = 'default',
+  Accent = 'accent',
+}
+
+export const StyledCard = styled('div', {
+  backgroundColor: '$neutral1',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'flex-start',
+  padding: '$lg',
+  width: 'max-content',
+  height: 'max-content',
+  boxShadow: '$shadows$card',
+  borderRadius: '$radii$lg',
+  variants: {
+    color: {
+      default: {
+        backgroundColor: '$defaultSurface',
+        color: '$defaultContrast',
+      },
+      accent: {
+        backgroundColor: '$colors$neutral2',
+        color: '$contrast',
```

Comment:
I don't believe this is a color defined in the theme?

Reply:
I've changed it into "Negative", but don't really see a color for that meaning in any other components

Reply:
I didn't mean the name of the variant! I meant "$contrast" is not a defined color in the theme. If you go to `styles/colors` you can see all of the available color tokens